modules/hivemq-edge-module-file/src/main/java/com/hivemq/edge/adapters/file/FilePollingProtocolAdapter.java

```diff
@@ -50,12 +47,14 @@ public class FilePollingProtocolAdapter implements PollingProtocolAdapter<FileTo
     private final @NotNull ProtocolAdapterInformation adapterInformation;
     private final @NotNull ProtocolAdapterState protocolAdapterState;
     private final @NotNull List<FileToMqttMapping> pollingContext;
+    private final @NotNull List<Tag> tags;
 
     public FilePollingProtocolAdapter(
             final @NotNull ProtocolAdapterInformation adapterInformation,
             final @NotNull ProtocolAdapterInput<FileAdapterConfig> input) {
         this.adapterInformation = adapterInformation;
         this.adapterConfig = input.getConfig();
+        this.tags = input.getTags();
```

Comment:
if the tags are stored at the beginnig, we wont see updates in them during the lifetime of the adapter. Is the adapter getting deleted on tag update?

Reply:
Yep, that's what we talked about last week :)
For now every change is destructive.